ci/docker/Dockerfile.build.ubuntu_base_cpu

```diff
@@ -25,13 +25,13 @@ WORKDIR /work/deps
 
 RUN apt-get update && apt-get -y install sudo
 
-ARG USER_ID=os.getuid()
-
+ARG USER_ID=0
+ARG GROUP_ID=0
 COPY install/ubuntu_adduser.sh /work/
 RUN /work/ubuntu_adduser.sh
 
-COPY install/ubuntu_runas_sudo.sh /work/
-RUN /work/ubuntu_runas_sudo.sh
+copy install/ubuntu_runas_sudo.sh /work/
```

Comment:
Why'd you lowercase this one?

Reply:
seems an editor mistake, will fix. Thanks for catching.